pkg/cli/image/mirror/mirror.go

```diff
@@ -601,18 +606,17 @@ func (o *MirrorImageOptions) plan() (*plan, error) {
 								// upload all the blobs
 								srcBlobs := srcRepo.Blobs(ctx)
 
-								// upload each manifest
-								for _, srcManifest := range srcManifests {
+								addBlobsForManifest := func(srcManifest distribution.Manifest) {
```

Comment:
Is the reason of this change for `addBlobsForManifest(srcManifest)`?

Reply:
Yes, so we can reuse the blob code for the parent and the children. The other option was to make a new slice with the parent and the children combined, but that would require a new allocation and I thought this was easier.